src/procedures/proc_functions.c

```
@@ -0,0 +1,145 @@
+    return NULL;
+}
+
+ProcedureResult Proc_FunctionsFree
+(
+	ProcedureCtx *ctx
+) {
+	// clean up
+	if(ctx->privateData) {
+		ProcFunctionsPrivateData *pdata = ctx->privateData;
+		raxStop(&pdata->iter);
+		array_free(pdata->output);
```

cppcheck

[error] Common realloc mistake

(memleakOnRealloc)